experimental/add_to_app/flutter_module_using_plugin/lib/main.dart

```diff
@@ -0,0 +1,134 @@
+// Copyright 2019 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:io';
+
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:path_provider/path_provider.dart' as paths;
+
+void main() => runApp(MyApp());
+
+class MyApp extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: 'Flutter Module Title',
+      routes: {
+        '/': (context) => FullScreenView(),
+        '/mini': (context) => MiniView(),
+      },
+    );
+  }
+}
+
+class FullScreenView extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text('A Full-screen Flutter View'),
```

Comment:
I would have assumed the word plugin would appear somewhere in this screen.

Reply:
Does the additional button in the plugin version do enough?

Reply:
I'd try to go with some pattern with these tittles. If the other module's title is A Full-screen Flutter View, maybe call this something that describes the persistence support.

Reply:
Makes sense. I'll update.